### ISSUE TITLE
Create new release v1.3.0

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/privacy-pioneer#595]
Given that we have the watchlist notification fix (#560) and also [updated pre-processing](https://github.com/privacy-tech-lab/privacy-pioneer-web-crawler/issues/9#issuecomment-2422754418), we should create a new release once the latter is done.

@atlasharry, feel free to go ahead with the release creation here on GitHub as well as the deployment on the Firefox Add-on store as you see fit. (We need to update the version number in the manifest to v1.3.0).

**Comment by atlasharry** [comment]
Still waiting for a release. Patton and I discussed and think it would be better to make this release after we have finished the second crawl.

**Comment by SebastianZimmeck** [comment]
Yes, that works. In either case, the release should have the exact codebase of the crawl.

**Comment by SebastianZimmeck** [comment]
Once #592 and #593 are fixed, we should create a new release.

**Comment by SebastianZimmeck** [comment]
As discussed in today's meeting, we will do some manual testing based on the current development version of Privacy Pioneer. If everything looks good, @atlasharry will create a release v1.3.0 here on GitHub, and I will deploy it on the Firefox Add-on Store.

So, @atlasharry and @PattonYin once you finalized your manual testing, please let me know.

**Comment by atlasharry** [comment]
@PattonYin I have done some testing on the version you fixed. Unfortunately I think this fix leads to some functional crash and maybe we need some further refinement. 
Currently, the back bottom in the detailed view of each tracking category would lead the extension to crash, in addition, the loading animation seems to be incorrect. The error in the console suggests this error is still related to reactDom: `Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot [r](webpack://privacy-pioneer/node_modules/react-dom/cjs/react-dom.development.js?61bb)`. I have check all different earlier versions and all other versions before April 16 are fine. 
@PattonYin Can you take a look at this issue to see if your fix is correct?

![Image](https://github.com/user-attachments/assets/742cbeea-cfc6-4c7a-9042-93295c6edf6b)

![Image](https://github.com/user-attachments/assets/4f40f4e8-ec42-45b8-8661-34b14d78d577)

I have tested the code
![Image](https://github.com/user-attachments/assets/7bd9864b-69be-41ff-8e03-25a6c1b30076)

**Comment by SebastianZimmeck** [comment]
@PattonYin, can you look into [what @atlasharry is pointing out](https://github.com/privacy-tech-lab/privacy-pioneer/issues/595#issuecomment-2918219516)?

**Comment by SebastianZimmeck** [comment]
Thanks, @PattonYin!

**Comment by SebastianZimmeck** [comment]
> Could you and @atlasharry run the extension now again to test if it works as expected?

@PattonYin and @atlasharry, could you check?

**Comment by PattonYin** [comment]
Yes, we've ran the extension to test. @atlasharry has pointed out some issues and I've fixed those with the commits last week. I also found some other bugs in the tutorial section, and I'm currently working on fixing them.

**Comment by SebastianZimmeck** [comment]
@atlasharry and @PattonYin, it looks like you fixed the bugs now, right? If so, can you test the extension and, if everything works now, create a release?

**Comment by atlasharry** [comment]
Yes! We will perform an comprehensive test on the extension and see if everything works fine! If so, we will create a new release.

**Comment by SebastianZimmeck** [comment]
Thank you, @atlasharry! And all the best for your filming! That is exciting!

@PattonYin, can you say?

**Comment by PattonYin** [comment]
I remember that I solved this, but I will double check by this weekend!

**Comment by PattonYin** [comment]
I can confirm that the earlier bugs with the tutorial UI have been resolved, and all tests have passed!
